portfolio/dash/rcx_callbacks.py

```
@@ -25,245 +25,267 @@ def _api(method: str, path: str, body: dict | None = None) -> dict:
-                    "name": name or site_id,
-                    "base_url": url,
-                    "username": user or "agent",
-                    "password": password or "",
-                    "auth_type": "password",
-                },
-            )
-            return f"Saved Edge {site_id}."
-        except Exception as exc:
-            return str(exc)
+    def refresh_edges(_tab, save_n, _remove_clicks, _interval_n, current_site, form_name, form_url, revision):
+        edges, api_err = _api_or_local_edges()
```

Ruff (0.15.15)

[warning] Unpacked variable `api_err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)